apps/src/templates/codeReviewGroups/CodeReviewGroupsStatusToggle.jsx

```diff
@@ -80,12 +83,12 @@ export const UnconnectedCodeReviewGroupsStatusToggle = CodeReviewGroupsStatusTog
 
 export default connect(
   state => ({
-    codeReviewExpiresAt: state.sectionData.section.codeReviewExpiresAt,
-    sectionId: state.sectionData.section.id
+    codeReviewExpiresAt: selectedSection(state).codeReviewExpiresAt,
```

Comment:
Nit: seems unlikely you'd hit this, but could you end up trying to read properties of null here?

Reply:
This component is on the manage students page, a section needs to be selected to end up on this page, so no, I don't believe you'd be able to end up with a null selected section